### ISSUE TITLE
[Req] Working examples for commands

### ISSUE THREAD

**Issue by Rohitrajak1807** [arlonproj/arlon#203]
Adding an `Example` property to each `cobra.Command` to provide the end user with working example can simplify their understanding of Arlon

**Comment by ShaunakJoshi1407** [comment]
I am adding some examples as a part of this PR: https://github.com/arlonproj/arlon/pull/202
Do let me know if some additional examples can be added

**Comment by Rohitrajak1807** [comment]
We would need more detailed exsmples. Take a look at https://github.com/platform9/appctl/blob/62a82734cc269765472bbf44d134bfbe86d5a90b/cmd/appDeploy.go#L18

**Comment by ShaunakJoshi1407** [comment]
Yep that can be added for commands like `arlon cluster`, `arlon profile `and `arlon bundle` commands

**Comment by Rohitrajak1807** [comment]
> Yep that can be added for commands like `arlon cluster`, `arlon profile `and `arlon bundle` commands

It'd be better to do it separately with ready to use detailed examples as it's own PR, I see that #202 also adds install and verify commands too.

**Comment by ShaunakJoshi1407** [comment]
Yes I was about to change that PR to just make changes in install and verify for now. A separate PR can be raised for this

**Comment by cruizen** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Arlon-Team-6345a9610e79cb1603a53d69/issues/arlonproj/arlon/203?planning-poker) with Zenhub @bcle @jayanth-tjvrr @Rohitrajak1807 @ShaunakJoshi1407

**Comment by cruizen** [comment]
@ShaunakJoshi1407 , @Rohitrajak1807 since you have worked on this and related docs earlier, please add some comments on the scope, what examples exist vs what don't - and please assign yourself if you plan to take it up.

**Comment by Rohitrajak1807** [comment]
These examples should be as simple as copy paste and go
